lib/mix/lib/mix/tasks/test.ex

```diff
@@ -425,7 +425,9 @@ defmodule Mix.Tasks.Test do
       do_run(opts, args, files)
     else
       {files_in_apps_path, files_not_in_apps_path} =
-        Enum.split_with(files, &String.starts_with?(&1, "apps/"))
+        files
+        |> Enum.map(&String.trim_leading(&1, "./"))
```

Comment:
I would if instead we should do something like Path.expand and then Path.relative_to_cwd?